src/main/java/org/jitsi/jicofo/codec/JingleOfferFactory.java

```diff
@@ -449,6 +449,11 @@ private static void addAudioToContent(ContentPacketExtension content, OfferOptio
             {
                 PayloadTypePacketExtension red = addPayloadTypeExtension(rtpDesc, config.opus.red.pt(), "red", 48000);
                 red.setChannels(2);
+
+                // RFC 2198 fmtp line
+                // Indicates Opus payload type (111) as primary/secondary encoding for RED (112)
+                // fmtp:112 111/111
+                addParameterExtension(red, null, config.opus.pt() + "/" + config.opus.pt());
```

Comment:
What are the semantics of "111/111"? As far as I understand 2198 (`we define the format
   parameters to be a slash "/" separated list of RTP payload types`) these are just formats that are declared usable with RED, so just "111" should suffice. But I haven't read the RFC carefully.

Reply:
you mean that if there's just one then it should be used for both primary and secondary?

Reply:
I did it this way, because that's what Phippo's post mentions here https://groups.google.com/g/discuss-webrtc/c/5761etCrSuA

and also I was able to confirm that it's working with Chrome. Not sure about the other way around.

Reply:
I read it again now, and I think I understand what you're saying.